app/alloy.js

```diff
@@ -1,7 +1,9 @@
 import ActionBarHelper from 'actionbar';
+import { isiOSVersionOrGreater } from './lib/app-utils';
```

Comment:
this fails for me with "module not found". When I change it to this it will load the lib:

```suggestion
import { isiOSVersionOrGreater } from 'app-utils';
```